src/geometry.rs

```diff
@@ -1039,6 +1165,28 @@ mod tests {
         (faces, vertices, normals)
     }
 
+    fn tessellated_triangle() -> (Vec<(u32, u32, u32)>, Vec<Point3<f32>>) {
+        #[rustfmt::skip]
+            let vertices = vec![
```

Comment:
Not formatted correctly. The point of using `#[rustfmt::skip]` is to format the thing manually, if you know better than the formatter.